app/controllers/centers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CentersController < ApplicationController
+	def create
+		@center = Center.find(params[:center_id])
```

Comment:
the `create` action is meant to create an object. Here you are just looking for and redirecting to its details page